src/posts/2019-03-20-publish-to-npm.md

```diff
@@ -19,7 +19,7 @@ But publishing packages the way the industry does it? Not so simple. There are m
 
 ## Creating your first package
 
-This section is for you if you haven't published a package to npm before. Feel free to skip to the next section if you published one before.
+This section is for you if you haven't published a package to npm before. Feel free to skip to the next section if you have published one before.
```

Comment:
Can you please change this to `you've`? Thanks a ton!

Reply:
Sure @zellwk will do that.